.github/workflows/dev_on_pull_request_run_e2e_web.yml

```
@@ -0,0 +1,112 @@
+
+name: '[on_pr] Run E2E smoke tests on Web'
+
+on:
+  push:
+    branches:
+        - PC-35750
+
+permissions:
+  contents: read
+  id-token: write
+  pull-requests: write
+
+jobs:
+  run_e2e_web:
+    name: 'Smoke test Staging Web'
+    runs-on: macos-15
```

je ne serais pas étonné qu'ubuntu coute moins cher que mac, je ne sais pas si c'est le cas

```suggestion
    runs-on: ubuntu-24.04
```

---

C'est moins chère oui